ISISScientificComputing/autoreduce
# Resubmit missing runs from EoRM

* Functionality in `ArchiveExplorer` to locate RB numbers for a given run.
* Function in `HealthChecker` to resubmit data that has been missed by EoRM
* Revert change on master from starting ICAT connection in `__init__`. There are occasions when you may want to initialise the client without the connection starting. Additionally it broke ICATClinet until tests

## To Test
* Code review
* Review of tests (these tests should cater for all use cases please consider if they do)

*As this uses ICAT this is going to be very difficult to set up a valid test environment for. However you can attempt the following:*
* Create a database that has a run for an instrument that is 3 ICAT e.g. `POLARIS113665`
* Create a fake archive and add a 4 data files  e.g. `POLARIS113665,POLARIS113666,POLARIS113667,POLARIS113668`
* Add entries in the `summary.txt` for all these files
* start the EndOfRunMonitor
* Observe in the logs and via activemq that all of these runs are resubmit
